akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/Effect.scala

```diff
@@ -58,7 +58,7 @@ object Effect {
    * Stop this persistent actor
    * Side effects can be chained with `andThen`
    */
-  def stop[Event, State]: Effect[Event, State] = none.andThenStop()
+  def stop[Event, State](): Effect[Event, State] = none.thenStop()
```

Comment:
My thinking here is that even though the method is just returning a directive the end result is a side effect and thereby motivates the `()`. Compared to `none` and `unhandled` that don't result in any side effects (well, unhandled might log something)